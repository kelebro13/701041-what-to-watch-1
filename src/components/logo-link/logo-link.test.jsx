import renderer from "react-test-renderer";
import {MemoryRouter} from "react-router-dom";
import LogoLink from "./logo-link";

it(`renders property`, () => {
  const wrapper = renderer
    .create(<MemoryRouter><LogoLink/></MemoryRouter>)
    .toJSON();

  expect(wrapper).toMatchSnapshot();
});
