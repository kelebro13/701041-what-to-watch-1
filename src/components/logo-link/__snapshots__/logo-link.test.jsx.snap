// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<a
  className=""
  href="/"
  onClick={[Function]}
>
  <span
    className="logo__letter logo__letter--1"
  >
    W
  </span>
  <span
    className="logo__letter logo__letter--2"
  >
    T
  </span>
  <span
    className="logo__letter logo__letter--3"
  >
    W
  </span>
</a>
`;
